common/djangoapps/student/views.py

```diff
@@ -573,8 +574,8 @@ def dashboard(request):
     course_enrollments.sort(key=lambda x: x.created, reverse=True)
 
     # set progress for each course
-    for course_enrollment in course_enrollments:
-        course = store.get_course(course_enrollments[0].course_id)
+    for i, course_enrollment in enumerate(course_enrollments):
```

Comment:
No need for this. You can simply use course enrollment.course_id. It was a bug introduced by me.